cmd/ncproxy/run.go

```diff
@@ -35,13 +36,18 @@ type nodeNetSvcConn struct {
 var (
 	// Global mapping of network namespace ID to shim compute agent ttrpc service.
 	containerIDToShim = make(map[string]computeagent.ComputeAgentService)
+
+	// Global database for containerID to compute agent address
+	agentStore *computeAgentStore
```

Comment:
Any reason this can't just be a field on `ttrpcService`? We should avoid global state whenever possible (that would ideally also apply to the other globals here, though I realize they're already checked in).